hubmap/query_app/apps.py

```diff
@@ -37,8 +39,11 @@ def compute_dataset_hashes():
     hash_dict = {}
     try:
         for uuid in Dataset.objects.all().values_list("uuid", flat=True):
+            print(uuid)
```

Comment:
It's probably worth removing these before merging -- if they're useful for logging, might be better to use the standard Python logging package and centralize where these logging messages are sent.